docs/docs/api/disposer.md

```diff
@@ -13,8 +13,11 @@ title: .disposer
 .disposer(function(any resource, Promise usingOutcomePromise) disposer) -> Disposer
 ```
 
+A meta method used to specify the disposer method that cleans up a resource when using `Promise.using`.
 
-A meta method used to specify the disposer method that cleans up a resource when using undefined.
+Returns a Disposer object which encapsulates the scarce resource as well as the method to clean it up. 
```

Comment:
True, by itself it doesn't say enough. I'd like to expand on the kind of resources which are typically managed this way, but for now I think just removing it will suffice.